bee-message/src/output/mod.rs

```diff
@@ -111,4 +111,15 @@ impl Output {
             Self::Nft(output) => output.amount(),
         }
     }
+
+    /// Return the native tokens of an `Output`, if any.
+    pub fn native_tokens(&self) -> Option<Vec<NativeToken>> {
```

Comment:
I think I would return a slice here, if you need a Vec you can do it yourself but for most usecases a slice would be enough